packages/manager/apps/web/client/app/email-domain/upgrade/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,17 @@
+{
+  "email_domain_upgrade_title": "Changer d'offre",
+  "email_domain_upgrade_step_header_offer": "Offre",
+  "email_domain_upgrade_step_header_offer_description": "Sélectionnez l'offre désirée.",
+  "email_domain_upgrade_step_header_offer_select_placeholder": "Sélectionnez…",
+  "email_domain_upgrade_step_header_duration": "Durée",
+  "email_domain_upgrade_step_header_duration_description": "Sélectionnez la durée désirée.",
+  "email_domain_upgrade_step_header_contracts": "Contrats",
+  "email_domain_upgrade_next_button_label": "Suivant",
+  "email_domain_upgrade_upgrade_button_label": "Changer d'offre",
+  "email_domain_upgrade_cancel_button_label": "Annuler",
+  "email_domain_upgrade_contracts_accept": "Afin de confirmer votre demande, veuillez accepter les conditions d'utilisation.",
+  "email_domain_upgrade_in_progress": "Modification de l'offre de messagerie électronique en '{{ newOffer }}'",
+  "email_domain_upgrade_fetch_duration_error": "Une erreur s'est produite lors de l'obtention des détails de l'offre: {{message}}",
+  "email_domain_upgrade_error": "Impossible de modifier l'offre de messagerie: {{message}}",
+  "email_domain_upgrade_success": "Le changement d'offre par e-mail est en cours. <a href=\"{{ orderTrackURL }}\" target=\"_blank\" rel=\"noopener\"> Cliquez ici pour suivre la progression. </a>"
```

Comment:
The translation could be splitted : `Le changement d'offre par e-mail est en cours.` & `Cliquez ici pour suivre la progression.`.